addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -2360,22 +2360,20 @@ export class OdooEditor extends EventTarget {
         if (this._isResizingTable || !traversedNodes.some(node => !!closestElement(node, 'td') && !closestElement(node, '[data-oe-protected="true"]'))) {
             return false;
         }
-        let range;
-        if (selection.rangeCount > 1) {
-            // Firefox selection in table works with multiple ranges.
-            const startRange = getDeepRange(this.editable, {range: selection.getRangeAt(0)});
-            const endRange = getDeepRange(this.editable, {range: selection.getRangeAt(selection.rangeCount - 1)});
-            range = this.document.createRange();
-            range.setStart(startRange.startContainer, 0);
-            range.setEnd(endRange.startContainer, 0);
-        } else {
-            // We need the triple click correction only for a bug in firefox
-            // where it gives a selection of a full cell as tr 0 tr 1. The
-            // correction makes it so it gives us the cell and not its neighbor.
-            // In all other cases we don't want to make that correction so as to
-            // avoid flicker when hovering borders.
-            range = getDeepRange(this.editable, { correctTripleClick: anchorNode && anchorNode.nodeName === 'TR' });
+        if (this.isFirefox) {
+            if (selection.rangeCount > 1) {
+                this.historyResetLatestComputedSelection(true);
+            } else if (ev && closestElement(ev.target, 'table') === closestElement(selection.anchorNode, 'table') &&
+                    closestElement(selection.focusNode, 'td') !== closestElement(ev.target, 'td')) {
+                setSelection(selection.anchorNode, selection.anchorOffset, ev.target, 0);
+            }
         }
+        // We need the triple click correction only for a bug in firefox
```

Comment:
Yes, we need to call getDeepRange for Chrome also. I don't see any issue with the comment, as it indicates that a triple-click is only required when the selection moves to a TR, which, as far as I know, only occurs in Firefox.

However, we can structure the code like this to avoid confusion and can be able to remove comment:

```
let range;
if (this.isFirefox) {
    if (selection.rangeCount > 1) {
        this.historyResetLatestComputedSelection(true);
    } else if (
        ev &&
        closestElement(ev.target, 'table') === closestElement(selection.anchorNode, 'table') &&
        closestElement(ev.target, 'td') !== closestElement(selection.focusNode, 'td')
    ) {
        setSelection(selection.anchorNode, selection.anchorOffset, ev.target, 0);
    }
    range = getDeepRange(this.editable, { correctTripleClick: anchorNode && anchorNode.nodeName === 'TR' });
} else {
    range = getDeepRange(this.editable);
}

```